rules/CodingStyle/Rector/Property/InlineSimplePropertyAnnotationRector.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\CodingStyle\Rector\Property;
+
+use PhpParser\Comment\Doc;
+use PhpParser\Node;
+use PhpParser\Node\Stmt\ClassConst;
+use PhpParser\Node\Stmt\Property;
+use Rector\Core\Contract\Rector\ConfigurableRectorInterface;
+use Rector\Core\Rector\AbstractRector;
+use Rector\NodeTypeResolver\Node\AttributeKey;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\ConfiguredCodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+use Webmozart\Assert\Assert;
+
+/**
+ * @see \Rector\Tests\CodingStyle\Rector\Property\InlineSimplePropertyAnnotationRector\InlineSimplePropertyAnnotationRectorTest
+ */
+final class InlineSimplePropertyAnnotationRector extends AbstractRector implements ConfigurableRectorInterface
```

Comment:
Using `ConfigurableRectorInterface` will make default value `annotationsToConsiderForInlining` property `['@var', '@phpstan-var', '@psalm-var']` always replaced as it doesn't allow empty config. I think it should be better to have it empty value as default:

```php
private array $annotationsToConsiderForInlining = [];
```

If you want to be allow empty, you can implements `\Rector\Core\Contract\Rector\AllowEmptyConfigurableRectorInterface` instead.

Reply:
`AllowEmptyConfigurableRectorInterface` is perfect, thanks! :+1:

Reply:
when no `->configure()` call, `['@var', '@phpstan-var', '@psalm-var']` will be used, could you add new test for it with new different fixture directory, config, and test class for it? eg: `FixtureEmptyConfig`, `configure_rule_empty_config`, and `EmptyConfigureTest.php`  under `rules-tests/CodingStyle/Rector/Property/InlineSimplePropertyAnnotationRector/`

Reply:
I already have both test case with configure called and with the default configuration. I will just rename the namespaces as you have suggested. :+1: